redaxo/src/addons/structure/pages/linkmap.php

```diff
@@ -83,7 +83,7 @@ function insertLink(link,name){
 if ($category) {
     foreach ($category->getParentTree() as $parent) {
         $n = [];
-        $n['title'] = str_replace(' ', '&nbsp;', htmlspecialchars($parent->getName()));
+        $n['title'] = str_replace(' ', '&nbsp;', rex_escape($parent->getName()));
```

Comment:
müssen wir in den addons eine rex min-version anpassen, damit rex_escape() immer vorhanden ist?

Reply:
grad quergechecked, zumindest im mediapool müsste man die rex min version anheben

Reply:
Struktur steht bereits auf ^5.5
Beim Mediapool muss es angepasst werden, mache ich gerade noch hier im PR.

Reply:
ok, d.h. man brauch min rex5.4 damit man keinen bekannte sicherheitslücke mehr hat.
sollten wir im newsletter drauf hinweisen dass man die neuste version verwenden sollte

Reply:
Bei XSS im eingeloggten Bereich habe ich bisher nie die addons einzeln als Update rausgegeben, sprich diese MP-Version kommt sowieso nur zusammen mit 5.5.1, wenn ich es mache, wie bisher.
Auch in 5.5.0 hatten wir ja fehlende Escapings hinzugefügt, die man nur mit 5.5.0 bekommt.

Reply:
ok passt... wie gesagt.. sollten wir die empfehlung rausgeben dass man immer die neuste version verwenden sollte